website/docs/language/backend/azurerm.mdx

```diff
@@ -514,16 +514,14 @@ The following configuration options are supported:
 
 * `key` - (Required) The name of the Blob used to retrieve/store Terraform's State file inside the Storage Container.
 
-* `environment` - (Optional) The Azure Environment which should be used. This can also be sourced from the `ARM_ENVIRONMENT` environment variable. Possible values are `public`, `china`, `german`, `stack` and `usgovernment`. Defaults to `public`.
-
-* `endpoint` - (Optional) The Custom Endpoint for Azure Resource Manager. This can also be sourced from the `ARM_ENDPOINT` environment variable.
-
-~> **NOTE:** An `endpoint` should only be configured when using Azure Stack.
+* `environment` - (Optional) The Azure Environment which should be used. This can also be sourced from the `ARM_ENVIRONMENT` environment variable. Possible values are `public`, `china`  and `usgovernment`. Defaults to `public`.
```

Comment:
```suggestion
- `environment`: (Optional) Specifies the Azure environment to use. Specify one of the following values:
      - `public`, 
      - `china`  
      - `usgovernment`. 
      The default is `public`.
   You can set the `ARM_ENVIRONMENT` environment variable to configure this option.
```
Addresses grammar issues and writing style violations.

Reply:
Done